.github/workflows/kind-e2e-upgrade.yaml

```diff
@@ -48,10 +53,10 @@ jobs:
       KIND_CLUSTER_NAME: "kourier-integration"
 
     steps:
-    - name: Set up Go 1.18.x
+    - name: Set up Go 1.19.x
       uses: actions/setup-go@v2
```

Comment:
Midstream we went to action v4 for the bump to 1.19. Maybe also something to consider?